bundles/org.eclipse.ui.themes/plugin.xml

```diff
@@ -51,6 +51,24 @@
             label="%theme.win"
             os="win32">
       </theme>
+      <theme
+            basestylesheeturi="css/e4_preview_gtk.css"
+            id="org.eclipse.e4.ui.css.theme.e4_preview"
```

Comment:
IMO, it would be worth finding it a new fresh id now that wouldn't change later. More precisely, the id shouldn't mention "preview". So later, when we want it to be default, we just change the default value of the preference and that's all.
People who have switched to this theme already wouldn't see an issue (the theme would still be present), and people who have never changed the theme should also see the update as the default value of the preference would change.
Maybe something like "flatLight" since this themes gets rid of many gradients, lines and constrasts.

Reply:
If you plean to review the theme with id `org.eclipse.e4.ui.css.theme.e4_preview`, then you may introduce issues to people who have enabled it and then update. The selected theme would then be missing.
To be safe, it's either changes in the current theme (but that couldn't be completed), or a new theme with a sustainable new id. Everything in between can be error-prone.

Reply:
I tested this. If I have set the preview theme. Then start an eclipse with that workspace that does not have that theme it defaults to the "standard" light theme.